boards/arm/nrf5340_dk_nrf5340/nrf5340_cpunet_reset.c

```diff
@@ -39,6 +39,11 @@ static void remoteproc_mgr_config(void)
 	 * accesses by Network MCU will have Secure attribute set).
 	 */
 	NRF_SPU->EXTDOMAIN[0].PERM = 1 << 4;
+
+	/* Treat shared memory as BSS section and initialize it to zero. */
```

Comment:
Remove comment about BSS - it give false impression that the Shared memory is (always) part of the image; it is not, actually. "Initialize inter-processor shared memory block to zero" is all yo need to mention, imo.

Reply:
fixed